.github/workflows/test.yml

```diff
@@ -5,6 +5,11 @@ on:
     branches: [main]
   pull_request:
     branches: [main]
+    paths:
+      - "strawberry/**"
+      - "tests/**"
+      - "pyproject.toml"
+      - "poetry.lock"
```

Comment:
oh, shall we put the action itself so if we change it we also test it in the same PR?

Reply:
Oh yeah good idea